src/main/scala/mesosphere/marathon/core/launcher/package.scala

```diff
@@ -0,0 +1,7 @@
+package mesosphere.marathon.core
+
+/**
+  * This package contains the glue code between matching tasks to resource offers
+  * and actually launching the matched tasks.
+  */
+package object launcher
```

Comment:
What do you think, moving this documentation to the module (and getting rid of the package object)?